content/docs/1.6.0/deploy/install/install-with-helm.md

```diff
@@ -8,9 +8,7 @@ In this section, you will learn how to install Longhorn with Helm.
 ### Prerequisites
 
 - Each node in the Kubernetes cluster where Longhorn will be installed must fulfill [these requirements.](../#installation-requirements)
```

Comment:
```
- Kubernetes cluster: Ensure that each node fulfills the [installation requirements](../#installation-requirements).
```

Reply:
This is the exact wording in the "Install with Flux" and "Install with Fleet" files. I suggest we use the same here.